### ISSUE TITLE
Migrate patternslib pattern to pat-tocify

### ISSUE THREAD

**Issue by instification** [collective/collective.tocify#9]
Migrate existing pattern to `pat-tocify` package.
- [x] Use yeoman generator to create layout
- [x] Migrate tocify pattern
- [ ] Update collective.tocify to build resources from `pat-tocify`
## Further information

http://patternslib.com/pattern-styleguide/#main-content

The styleguide is incomplete and needs expanding. It's on my TODO list and I'll try to get to it ASAP.

One easy way to check for minimal compatibility is to copy the .jshint file from Patterns repo and put that in your own and then do a check.

Another easy way to get the pattern to adhere to conventions is to use the Yeoman generator to generate the package layout (you can then copy your pattern into the src dir).
https://www.npmjs.com/package/generator-patternslib

**Comment by allcaps** [comment]
@instification: Can you review: https://github.com/collective/pat-tocify ?

I used the latest: `yo patternslib pat-tocify`. As suggested by @thet via mail and available for all via https://www.npmjs.com/package/generator-patternslib

**Comment by instification** [comment]
@allcaps yes looking now. Annoyingly I already did this but didn't push it (was having the widget issue from #5 which you solved!!).

**Comment by allcaps** [comment]
Credits to Jens Klein who showed his jquery.ui folder loading during JS training. ;)

**Comment by instification** [comment]
@allcaps  just built pat-tocify and still getting the widget issue ( #5 ) perhaps I should create a new issue in the pat-tocify repo?
